docker-compose.yaml

```diff
@@ -1,3 +1,6 @@
+version: '3.4'
```

Comment:
The version field is [obsolete](https://docs.docker.com/compose/compose-file/04-version-and-name/#version-top-level-element-obsolete) and is no longer required.

Reply:
removed, thanks